test/kixi/integration/base.clj

```diff
@@ -117,28 +120,25 @@
                                :decode t/read-handlers}}))
 (defn wait-for-url
   ([url]
-   (wait-for-url url 10))
-  ([url tries]
-   (when (pos? tries)
-     (Thread/sleep 500)
-     (let [md (client/get url
-                          {:accept :transit+json
-                           :as :stream
-                           :throw-exceptions false
-                           :transit-opts {:encode t/write-handlers
-                                          :decode t/read-handlers}})]
-       (if (= 404 (:status md))
-         (recur url (dec tries))
-         md)))))
+   (wait-for-url url 50 nil))
+  ([url tries last-result]
+   (if (pos? tries)
+     (do
+       (Thread/sleep 100)
```

Comment:
should the sleep happen in a do in the if below so that it doesn't sleep 100 if it doesn't need to?